CHANGELOG.md

```diff
@@ -2,6 +2,59 @@
 
 ## [Unreleased]
```

Comment:
Changes are presented in unstructured way, I suggest structuring them using guidelines found in https://keepachangelog.com/en/1.1.0/#how

Reply:
CHANGELOG modified in accoradance to https://keepachangelog.com/en/1.1.0/#how